### ISSUE TITLE
command 'C_Cpp.Navigate' not found

### ISSUE THREAD

**Issue by hensz** [microsoft/vscode-cpptools#4469]
Hi there!

Since some time, the command C_Cpp.Navigate is not working anymore, I get the above mentioned error message. This occurs both with native VS Code (1.35.0) on my Linux box (SLES 12.3) with cpptools v0.26.0 and on my Windows VS Code (1.39.1) with cpptools v0.26.0 with using Remote Development.

The command worked fine since when I started to use VS Code. Even if the index was wrong, I used an alternative indexer or the compilecommands file was corrupt this command worked (perhaps not displaying anything helpful, but at least I got no error message).

I cannot tell when exactly the issue started to occur, and I don't know which further information I can provide to track down the root cause. So if you need something, just ask, I'll provide the required details.

Thanks,
Torsten

**Comment by bobbrow** [comment]
VS Code provides the same feature in the breadcrumbs bar now, so we removed our custom implementation.  You can use the `breadcrumbs.focusAndSelect` command to achieve the same purpose.

Thank you for using the C++ extension!

**Comment by hensz** [comment]
Great, thanks a lot. Just a short follow-up question: Is there a way to see these discontinued features/commands and the respective replacements?

**Comment by hensz** [comment]
By the way, I just noticed that the command focusAndSelect is not quite the same as Navigate. The advantage of Navigate is: At least on my machine focusAndSelect needs quite some time until all information is read so that I can jump to a certain location in the file (currently it takes > 30 seconds), whereas Navigate was available nearly instantly. I'm not sure how this was implemented, but I really miss this feature :-(
btw,  also the feature "Go to Symbol in File" is really slow on my machine, most likely it uses the same data than focusAndSelect.

Shall I open a different bug for this, since actually the "error" is indeed by design, but the user experience problem most likely is not?

PS: I'm using a rather powerful machine for VS Code: My machine has 48 Xeon cores (Skylake) and 384 GB RAM.
